extensions/amp-analytics/0.1/analytics-root.js

```diff
@@ -300,7 +302,7 @@ export class AnalyticsRoot {
         }
         for (let j = 0; j < nodeList.length; j++) {
           if (this.contains(nodeList[j])) {
-            elementArray.push(nodeList[j]);
+            this.checkElementDataVars(nodeList[j], elementArray, selector);
```

Comment:
nit: Can we make this a private method. 
Personally I would expect the `checkElementDataVars` to return a boolean based on the method name, and you won't need to pass the `elementArray` or `selector`. Not blocking.

Reply:
Changed the method.